### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Fatih120** [hylian-modding/OotO-VoiceTemplate#2]
It would be nice to know what to do.

**Comment by Nergana** [comment]
Agreed. I've been plugging away at this crap for two days and counting trying to get a single voice mod to work. I'm so cliose, I just keep getting a PushID/PopID or TreeNode/TreePop mismatch error.

**Comment by denoflionsx** [comment]
You make folders in the sounds folder based on the id you're aiming to replace. Ids are found in the [included text document](https://github.com/hylian-modding/OotO-VoiceTemplate/blob/main/src/OotO-VoiceTemplate/sounds/SoundID_Info.txt). 

You put .ogg files in those folders. That is all there is to it.
